[cloud-provider-yandex] Update terraform provider

### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

We use 0.45.1 version, now available [0.74.0](https://github.com/yandex-cloud/terraform-provider-yandex/releases/tag/v0.74.0)

Maybe, it will fix terraform panic when delete public IP 

```
panic: interface conversion: interface {} is nil, not string
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1: 
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1: goroutine 44 [running]:
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1:                                                                 ↵
github.com/yandex-cloud/terraform-provider-yandex/yandex.handleAddressNotFoundError(0x1a364e0, 0xc00090aa50, 0xc0002a48c0, 0xc00090aa50,   ↵
0x0)
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1:                                                                 ↵
	github.com/yandex-cloud/terraform-provider-yandex/yandex/resource_yandex_vpc_address.go:20 +0x135
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1:                                                                 ↵
github.com/yandex-cloud/terraform-provider-yandex/yandex.resourceYandexVPCAddressDelete(0xc0002a48c0, 0x1633b20, 0xc0009b8e00, 0x0, 0x0)
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1:                                                                 ↵
	github.com/yandex-cloud/terraform-provider-yandex/yandex/resource_yandex_vpc_address.go:289 +0x3a5
2022-05-13T19:41:30.559Z [DEBUG] plugin.terraform-provider-yandex_v0.45.1:                                                                 ↵
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc000355680, 0xc000293540, 0xc00073b940, 0x1633b20,
```

### Proposed Solution

Just update to new version

### Additional Information

_No response_

--- a/.github/ci_includes/terraform_versions.yml
+++ b/.github/ci_includes/terraform_versions.yml
@@ -19,6 +19,6 @@ TF_VSPHERE_TYPE: vsphere
 TF_VSPHERE_VERSION: 2.0.2
 TF_YANDEX_NAMESPACE: yandex-cloud
 TF_YANDEX_TYPE: yandex
-TF_YANDEX_VERSION: 0.45.1
+TF_YANDEX_VERSION: 0.74.0
 # </template: terraform_versions_envs>
 {!{- end -}!}

--- a/.github/workflows/build-and-test_dev.yml
+++ b/.github/workflows/build-and-test_dev.yml
@@ -73,7 +73,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Analog of Gitlab's "interruptible: true" behaviour.

--- a/.github/workflows/build-and-test_release.yml
+++ b/.github/workflows/build-and-test_release.yml
@@ -81,7 +81,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Analog of Gitlab's "interruptible: true" behaviour.

--- a/.github/workflows/deploy-web-stage.yml
+++ b/.github/workflows/deploy-web-stage.yml
@@ -91,7 +91,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
   # Analog of Gitlab's "interruptible: true" behaviour.

--- a/.github/workflows/deploy-web-test.yml
+++ b/.github/workflows/deploy-web-test.yml
@@ -91,7 +91,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
   # Analog of Gitlab's "interruptible: true" behaviour.

--- a/.github/workflows/e2e-aws.yml
+++ b/.github/workflows/e2e-aws.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/e2e-azure.yml
+++ b/.github/workflows/e2e-azure.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/e2e-gcp.yml
+++ b/.github/workflows/e2e-gcp.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/e2e-openstack.yml
+++ b/.github/workflows/e2e-openstack.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/e2e-static.yml
+++ b/.github/workflows/e2e-static.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/e2e-vsphere.yml
+++ b/.github/workflows/e2e-vsphere.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/e2e-yandex-cloud.yml
+++ b/.github/workflows/e2e-yandex-cloud.yml
@@ -96,7 +96,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Note: no concurrency section for e2e workflows.

--- a/.github/workflows/schedule-cleanup.yml
+++ b/.github/workflows/schedule-cleanup.yml
@@ -71,7 +71,7 @@ env:
   TF_VSPHERE_VERSION: 2.0.2
   TF_YANDEX_NAMESPACE: yandex-cloud
   TF_YANDEX_TYPE: yandex
-  TF_YANDEX_VERSION: 0.45.1
+  TF_YANDEX_VERSION: 0.74.0
   # </template: terraform_versions_envs>
 
 # Always run a single job at a time.

